lusSTR/scripts/marker.py

```diff
@@ -388,12 +388,18 @@ def convert(self):
         else:
             if "GGGCTGCCTA" in self.uas_sequence:
                 break_point = self.uas_sequence.index("GGGCTGCCTA") + 10
-            else:
+                bracketed_form = (
+                    f"{collapse_repeats_by_length(self.uas_sequence[:break_point], 4)} "
+                    f"{collapse_repeats_by_length(self.uas_sequence[break_point:], 4)}"
+                )
+            elif "TTTT" in self.uas_sequence:
                 break_point = self.uas_sequence.index("TTTT") + 14
-            bracketed_form = (
-                f"{collapse_repeats_by_length(self.uas_sequence[:break_point], 4)} "
-                f"{collapse_repeats_by_length(self.uas_sequence[break_point:], 4)}"
-            )
+                bracketed_form = (
+                    f"{collapse_repeats_by_length(self.uas_sequence[:break_point], 4)} "
+                    f"{collapse_repeats_by_length(self.uas_sequence[break_point:], 4)}"
+                )
+            else:
+                bracketed_form = ""
```

Comment:
Once again ran into an issue with this marker- this time, the sequence seemed to be off by 10s of bases on one side (e.g. the sequence started ~50 bases early on the 5' side thus not containing the entire repeat region and was missing the ```GGGCTGCCTA``` and the ```TTTT``` sequences the code originally searched for). Now the code just basically drops these sequences- they are garbage.

Reply:
Rebecca emphatically declares that these sequences cannot be redeemed.